config-model/src/main/java/com/yahoo/vespa/model/content/ClusterControllerConfig.java

```diff
@@ -122,79 +112,83 @@ public void getConfig(FleetcontrollerConfig.Builder builder) {
+        private final Optional<Double> minStorageUpRatio;
+        private final Optional<Integer> minSplitBits;
+        private final Optional<Integer> maxGroupsAllowedDown;
+
+        ClusterControllerTuningBuilder(ModelElement tuning,
+                                       Optional<Double> minNodeRatioPerGroup,
+                                       Optional<Integer> bucketSplittingMinimumBits,
+                                       boolean maxGroupsAllowedDown,
+                                       int numberOfLeafGroups) {
+            this.minSplitBits = bucketSplittingMinimumBits;
+            this.minNodeRatioPerGroup = minNodeRatioPerGroup;
+            if (tuning == null) {
+                this.initProgressTime = Optional.empty();
+                this.transitionTime = Optional.empty();
+                this.maxPrematureCrashes = Optional.empty();
+                this.stableStateTimePeriod = Optional.empty();
+                this.minDistributorUpRatio = Optional.empty();
+                this.minStorageUpRatio = Optional.empty();
+                this.maxGroupsAllowedDown = Optional.empty();
+            }
+            else {
+                this.initProgressTime = Optional.ofNullable(tuning.childAsDuration("init-progress-time"));
+                this.transitionTime = Optional.ofNullable(tuning.childAsDuration("transition-time"));
+                this.maxPrematureCrashes = Optional.ofNullable(tuning.childAsLong("max-premature-crashes"));
+                this.stableStateTimePeriod = Optional.ofNullable(tuning.childAsDuration("stable-state-period"));
+                this.minDistributorUpRatio = Optional.ofNullable(tuning.childAsDouble("min-distributor-up-ratio"));
+                this.minStorageUpRatio = Optional.ofNullable(tuning.childAsDouble("min-storage-up-ratio"));
+                this.maxGroupsAllowedDown = maxGroupsAllowedDown(tuning, maxGroupsAllowedDown, numberOfLeafGroups);
+            }
         }
-    }
 
-    private ClusterControllerTuning build() {
-        return new ClusterControllerTuning(initProgressTime,
-                                           transitionTime,
-                                           maxPrematureCrashes,
-                                           stableStateTimePeriod,
-                                           minDistributorUpRatio,
-                                           minStorageUpRatio,
-                                           maxGroupsAllowedDown,
-                                           minNodeRatioPerGroup,
-                                           minSplitBits);
-    }
 
-}
+        private static Optional<Integer> maxGroupsAllowedDown(ModelElement tuning, boolean allowMoreThanOneContentGroupDown, int numberOfLeafGroups) {
+            var minGroupsUpRatio = tuning.childAsDouble("min-group-up-ratio");
+
+            if (minGroupsUpRatio != null) {
+                if (minGroupsUpRatio < 0.01 || minGroupsUpRatio > 1)
```

Comment:
Why is  `0 < minGroupsUpRatio < 0.01`  disallowed?

Reply:
Will fix at the same time as changing the name